src/lib/components/Dialog/content.svelte

```diff
@@ -0,0 +1,48 @@
+<script lang="ts" context="module">
+	import { focusTrap } from '$lib/internal';
+	import { dismissable } from '$lib/internal/actions/dismissable';
+	import { removeScroll } from '$lib/internal/actions/removeScroll';
+	import type { BaseProps } from '$lib/types';
+	import { getDataState } from './internal/helpers';
+	import { getDialogRootContext } from './root.svelte';
+
+	export type DialogContentProps = BaseProps<'div'> & {
+		onPointerDownOutside?: (event: CustomEvent<{ originalEvent: MouseEvent }>) => void;
```

Comment:
I'm curious why you chose properties for this, and not sveltes dispatched events?